src/pages/contributors/yaroslav-afenkin.adoc

```diff
@@ -0,0 +1,96 @@
+By the time I finished school, I got to a pretty decent level, which helps a lot in the industry that I'm working in.
+I then entered university in Ukraine, but after studying here for two years, it wasn't as good as I had hoped.
+I decided to move to Warsaw, Poland and continued my studies there.
+With one year left to study, I found a job working as a junior software engineer.
+Initially, I was hired to work on some insurance stuff.
+They were using a suite was called Guidewire.
+However, I was lucky enough that there was no Guidewire project for me.
+
+As a result of this, I landed on a project with a very heavy Java stack, Java Spring, and Angular in the front end.
+That really helped a lot and I feel like I won a lottery there because that's how I developed most of my Java and Javascript skills.
+I ended up working there for three years while I finished university and stayed in Poland.
+Then Covid happened and for various reasons, I eventually moved back to Ukraine.
+This meant that I had to leave my job, but I found a new job and ended up in CloudBees.
+Initially there was this little project where we were cleaning up some security technical debt with some other folks.
+As my project was coming to an end, we were asked where we wanted to go and I had a preference for security.
+This is how I ended up in the Jenkins security team in CloudBees and that's how I gained most of my Jenkins skills that I have right now.
+This also helped a lot when working on the recent link:https://www.jenkins.io/blog/2024/10/04/content-security-policy-grant/[CSP project in Jenkins].
+
+== How long have you been working on Jenkins?
+
+It's probably worth mentioning that I used Jenkins in my first job.
+That was my initial exposure to Jenkins, despite not using it too heavily.
+We basically had CI/CD and no code review stack, so we were using Garrett and Jenkins.
+I was just a basic user checking build logs and performing more basic actions.
+But there were some micro projects where I had to do performance tests for a software that we were using.
+I used Jenkins for it, despite not being sure it was optimal for my purposes, but that's one notable thing I can think of. 
+
+Other than that, I wasn't necessarily a heavy user of Jenkins.
+Most of my work has been done on Jenkins instead of through Jenkins.
+It's not like I need Jenkins during my daily life outside of the job.
+I'm trying to keep the balance between spending time in front of a PC doing work stuff and my spare time.
+So, I'm just trying to relax a bit in my spare time and keep work at work.
+
+== Was there any other aspects of Jenkins that interested you when you got started with it or that you found a little bit more useful?
+
+There was one guy who is also a member of the community, Damien Strapanik.
+He sat right behind me and he was our de-facto admin since he was doing most of the admin tasks.
```

Comment:
FWIW
```suggestion
He sat right behind me and he was our de-facto admin since he was doing most of the admin tasks for our project.
```

(we still had de jure admins who were doing admin work on the rest of the instance, as there were other projects, teams, etc.)

Reply:
Ah, okay that makes sense. Thanks very much!